package.json

```diff
@@ -15,6 +15,7 @@
     "browserify-middleware": "^5.0.2",
     "express": "^4.12.4",
     "mithril": "^0.2.0",
+    "orientdb": "^1.3.0",
```

Comment:
I think this is unrelated. You can run `npm uninstall orientdb --save` to remove it